### ISSUE TITLE
Can't locate DBIx/Class/TimeStamp.pm

### ISSUE THREAD

**Issue by eserte** [wreis/DBIx-Class-EncodedColumn#14]
t/author/timestamp_order.t fails if DBIx::Class::TimeStamp is not installed:
```
...
{UNKNOWN}: Can't locate DBIx/Class/TimeStamp.pm in @INC (you may need to install the DBIx::Class::TimeStamp module) (@INC contains: ...
 .) at /home/cpansand/.cpan/build/2019091705/Class-C3-Componentised-1.001002-0/blib/lib/Class/C3/Componentised.pm line 146.
 at /usr/local/lib/perl5/5.30/base.pm line 137.
Compilation failed in require at /usr/local/lib/perl5/5.30/base.pm line 137.
	...propagated at /usr/local/lib/perl5/5.30/base.pm line 159.
BEGIN failed--compilation aborted at /home/cpansand/.cpan/build/2019091705/DBIx-Class-EncodedColumn-0.00018-0/t/author/../lib/DigestTest/Schema/WithTimeStampChild.pm line 6.
Compilation failed in require at /home/cpansand/.cpan/build/2019091705/Class-C3-Componentised-1.001002-0/blib/lib/Class/C3/Componentised.pm line 146. at /home/cpansand/.cpan/build/2019091705/Class-C3-Componentised-1.001002-0/blib/lib/Class/C3/Componentised.pm line 151
t/author/timestamp_order.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
...
```

**Comment by eserte** [comment]
t/author/timestamp.t is also affected.

**Comment by wreis** [comment]
That is declared as [dep](https://github.com/wreis/DBIx-Class-EncodedColumn/blob/master/cpanfile#L23) for author tests.

**Comment by eserte** [comment]
Then it's not properly skipped for some reason. At least the CPAN Testers Matrix does not look good, a lot of failures listed here: http://matrix.cpantesters.org/?dist=DBIx-Class-EncodedColumn+0.00018

**Comment by wreis** [comment]
Fixed in 0.00019
